src/script/interpreter.cpp

```diff
@@ -301,6 +301,10 @@ bool EvalScript(std::vector<std::vector<unsigned char> >& stack, const CScript&
                 opcode == OP_RSHIFT)
                 return set_error(serror, SCRIPT_ERR_DISABLED_OPCODE); // Disabled opcodes.
 
+            // OP_CODESEPARATOR in non-segwit transaction is invalid even in an unexecuted branch
```

Comment:
nit: would have pushed that in the switch case.

Reply:
The switch case wouldn't get reached if it's in an unexecuted branch...?

Reply:
Yep, disregard what I said, it would not have worked.